src/Internal/ParameterGenerators.php

```
@@ -0,0 +1,195 @@
+            // sends the reader hunting for which parameter asked for it.
+            throw new \InvalidArgumentException(sprintf(
+                'Cannot generate %s: it is not instantiable%s; pass an override',
+                $class,
+                self::via($chain),
+            ));
+        }
+
+        $constructor = $reflection->getConstructor();
+
+        if (!$constructor instanceof \ReflectionMethod) {
+            return Gen::constant([]);
```

PHPStan (2.2.7)

[error] Method Rasuvaeff\PropertyTesting\Internal\ParameterGenerators::forConstructor() should return Rasuvaeff\PropertyTesting\ArbitraryInterface<array<string, mixed>> but returns Rasuvaeff\PropertyTesting\Arbitrary\ConstantArbitrary<array{}>.
Template type TValue on class Rasuvaeff\PropertyTesting\ArbitraryInterface is not covariant. Learn more: https://phpstan.org/blog/whats-up-with-template-covariant

(return.type)